src/BaseSchema.hack

```diff
@@ -21,4 +21,20 @@ abstract class BaseSchema {
     ): Awaitable<ValidFieldResult<?dict<string, mixed>>> {
         return new ValidFieldResult(null);
     }
+
+    <<__MemoizeLSB>>
+    public static function getQueryType(): Types\ObjectType {
```

Comment:
Obviously this is lame. I just wanted to minimize the diff. I think @mwildehahn 's introspection PR already fixes these TODOs.